tutorials/cloud-iot-enviro-board-workshop/index.md

```diff
@@ -179,10 +181,9 @@ In this section, you configure the Raspberry Pi to send sensor data to IoT Core.
 ```bash
 export PROJECT_ID=your-gcp-project-id
 ```
-1. In Raspberry Pi shell set the your GCP project id in the device application configuration file.
-```bash
-sed -i -e 's/<replace with project id>/'"${PROJECT_ID}"'/g' $HOME/enviro-board/cloud_config.ini
-```
+1.  In the Raspberry Pi shell set your GCP project ID in the device application configuration file.
+
+        sed -i -e 's/&lt;replace with project id&gt;/'"${PROJECT_ID}"'/g' $HOME/enviro-board/cloud_config.ini
```

Comment:
replacing < > with &lt; &gt;/ break the shell command and they are not rendered as it is in the tutorial. It need to be <> for the command to work

Reply:
They will be rendered as < and > in the published tutorial. We can't use left angle brackets at all in the source files for these tutorials, since the security sanitizer strips anything that resembles HTML.